pkg/scaffold/v2/controller/controller_suitetest.go

```diff
@@ -32,16 +32,30 @@ var _ input.File = &ControllerSuiteTest{}
 type ControllerSuiteTest struct {
 	input.Input
 
-	// Resource is the resource to scaffold the controller_kind_test.go file for
+	// Resource is the Resource to make the Controller for
 	Resource *resource.Resource
+
+	// ResourcePackage is the package of the Resource
+	ResourcePackage string
```

Comment:
We just removed this field because it is not needed, leave the call to `GetResourceInfo` in the `Update` method as it was, you have access to `a.Multigroup`over there.

Reply:
See the line https://github.com/kubernetes-sigs/kubebuilder/pull/1198/files/360198f2079528274d762ad755290454f38047bd#diff-ec3d96c196a007869f26993a5a8a7492L122 which was removed and the line https://github.com/kubernetes-sigs/kubebuilder/pull/1198/files/360198f2079528274d762ad755290454f38047bd#diff-ec3d96c196a007869f26993a5a8a7492L122 which was changed. 

Closing as sorted out.

Reply:
Again, no, dont close.
This field is not required and was removed in a previous PR.
Keeo it as it was and use the multigroup from uodate instead of getinput